src/lib/ChargeStation/eventHandlers/ocpp-20/send-authorize.ts

```diff
@@ -11,10 +9,10 @@ const sendAuthorize: ChargeStationEventHandler<
 > = async ({ chargepoint, session }) => {
   await sleep(1000);
 
-  await chargepoint.writeCall(
-    'Authorize',
-    { idToken: { idToken: session.options.uid, type: 'ISO14443' } },
-    session
+  chargepoint.writeCall(
+      'Authorize',
+      {idToken: {idToken: session.options.uid, type: 'ISO14443'}},
+      session
```

Comment:
Check your formatter, it inserts 4 spaces, and removes padding from object literal.